cmd/tailscaled/tailscaled.go

```diff
@@ -78,6 +78,7 @@ var args struct {
 	port           uint16
 	statepath      string
 	socketpath     string
+	kubesecret     string
```

Comment:
we've got an odd mix of styles here, but given the flag is `kube-secret`, I think this may as well be `kubeSecret`